Composer/packages/ui-plugins/virtual-assistant-creation/src/components/provisionSummaryPage.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+import { RouterPaths } from '../shared/constants';
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+// -------------------- BotTypeTile -------------------- //
+interface ProvisionSummaryPageProps extends RouteComponentProps {
+  onDismiss: () => void;
+  onSubmit: () => void;
+}
+
+export const ProvisionSummaryPage: React.FC<ProvisionSummaryPageProps> = (props) => {
+  const { onDismiss, onSubmit } = props;
+
+  const columns: IColumn[] = [
+    {
+      key: 'column1',
+      name: formatMessage('Resource'),
+      fieldName: 'Resource',
+      minWidth: 50,
+      maxWidth: 200,
+      isResizable: false,
+      isMultiline: false,
+    },
+    {
+      key: 'column2',
+      name: formatMessage('Notes'),
+      fieldName: 'Notes',
+      minWidth: 300,
+      maxWidth: 500,
+      isResizable: false,
+      isMultiline: true,
+    },
+  ];
+
+  const items = [
```

Comment:
Please move this outside of the component.

Reply:
Note that the `formatMessage` calls will need to be invoked at render time. It's fine to move this out of the component, but those properties need to be functions instead. ex: `() => formatMessage('...')`

Reply:
I moved `items` outside of the component and turned it into a function called `provisioningDetailItems` which returns the array of content with strings leveraging the formatMessage function.